src/Spago/Command/Publish.purs

```diff
@@ -105,14 +100,12 @@ publish _args = do
   -- As first thing we run a build to make sure the package compiles at all
   runSpago
     -- We explicitly list the env fields because `Record.merge` didn't compile.
-    { getManifestFromIndex: env.getManifestFromIndex
-    , getMetadata: env.getMetadata
-    , workspace: env.workspace { selected = Just selected }
-    , logOptions: env.logOptions
+    -- FIXME: I think we can make it work
```

Comment:
FIXME?

Reply:
I was hoping to fix this one too, but I can't find a good way to make it work. So I'll think about it some more or take out this FIXME